README.md

```diff
@@ -116,6 +116,10 @@ Options:
              set this lower.
   --debug    Provide additional debug output. Use --debug=2 for more
              verbose output
+  --minSeverity { warning | error }
+             Ignore SWCs below the designated level
+  --swcBlacklist { ["SWC-103"] | ["SWC-101", "SWC-111"] | ...}
+             Ignore a specific SWC or list of SWCs.
```

Comment:
Can we update the arguments to `--min-severity` and `--swc-blacklist` for consistency with the other arguments?

And can you also add these to the `--help` menu? You can find that in around line 75 of `helpers.js`.

Reply:
makes sense! I've since added these so sorry for the delay there. I was also thinking of adding some improvements to make the SWCs easier to manage. i.e. just a comma separted list of codes for the cli rather than an array and having to prefix SWC for every one. An additional consideration could be concatenating various blacklists rather than simply overwriting with a single value.

As that all seemed more of a UX consideration, I held off for now. Let me know if you have any other concerns :+1: